### PR TITLE
=rem #17084 Skip test if 127.0.1.1 is not available

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #17084

**Comment by drewhk** [comment]
LGTM

File: akka-remote/src/test/scala/akka/remote/transport/netty/NettyTransportSpec.scala [hunk 1]
```diff
@@ -145,7 +145,14 @@ trait BindBehaviour { this: WordSpec with Matchers ⇒
 
     s"bind to specified $proto address" in {
       val address = SocketUtil.temporaryServerAddress(address = "127.0.0.1", udp = proto == "udp")
-      val bindAddress = SocketUtil.temporaryServerAddress(address = "127.0.1.1", udp = proto == "udp")
+      val bindAddress =
+        try SocketUtil.temporaryServerAddress(address = "127.0.1.1", udp = proto == "udp")
+        catch {
+          case e: java.net.BindException ⇒
+            println(s"skipping test due to [${e.getMessage}], you probably have to use `ifconfig lo0 alias 127.0.1.1`")
```
Comment (drewhk) [inline akka-remote/src/test/scala/akka/remote/transport/netty/NettyTransportSpec.scala]: you can call info() here (that makes it a test output)

Comment (patriknw) [inline akka-remote/src/test/scala/akka/remote/transport/netty/NettyTransportSpec.scala]: ah, great

**Comment by patriknw** [comment]
PLS BUILD

**Comment by patriknw** [comment]
replaced the println with info, will merge after pr-validation

**Comment by ktoso** [comment]
LGTM

**Pull request merged**